docker/.env.testnet.example

```diff
@@ -57,6 +57,15 @@ INJ_PEGGO_STATSD_ADDR=""
 GIT_TESTNET_TAG="v1.10-1677061209"
 GIT_MAINNET_TAG="v1.8.0-1668679102"
 
+# sidecar env #
+SIDECAR_IMAGE_VERSION=local
+SIDECAR_HEALTH_BLOCK_DIFF="30"
+SIDECAR_SENTRY_URL="https://testnet.sentry.tm.injective.network"
```

Comment:
Lets update those env 
`HORACLE_URL="https://k8s.mainnet.horacle.injective.network/height/latest"` Missing 
`SENTRY_URL=http://injective-core:26657` Its clear for people to understand it should point to local chain node